stockAndPrices/services/app/main.py

```diff
@@ -16,12 +16,39 @@ def home():
 
 @app.route('/login', methods=['POST', 'GET'])
 def login():
+    rtn = ""
+    db_host = request.args.get('dbHost')
+    db_port = request.args.get('dbPort')
+    try:
+        client = MongoClient(db_host, int(db_port))
+        client.server_info()
+    except:
+        rtn = "Connection Error! Please provide valid dbHost and dbPort as query strings and make sure your" \
+              " mongoDB server is running."
+        return f"<h1>{rtn}<h1>"
+
+    sample_db = client.sampleDB
+    dev_col = sample_db.developer
     if request.method == "GET":
-        return "<h1>Login Page</h1>"
+        if dev_col.find_one({"first": "Hello"}) is None:
+            dev_col.insert_one({"first": "Hello", "last": "World"})
+        document = dev_col.find_one({"first": "Hello"})
+        rtn = document["first"] + " " + document["last"]
+        return f"<h1>Login Page {rtn}<h1>"
     else:
-        user = request.form.get('username')
-        #pwd = request.form.get('password')
-        return f"<h1>Welcome {user}!</h1>"
+        user = request.form['user_name']
```

Comment:
Can we change user_name to username? I believe we are using username now.
@lilianahechen